x-pack/plugins/lens/public/drag_drop/drag_drop.scss

```
@@ -6,6 +6,21 @@
   transition: background-color $euiAnimSpeedFast ease-in-out, border-color $euiAnimSpeedFast ease-in-out;
 }
 
+.lnsDragDrop_ghost {
+  @include lnsDraggable;
+  border: $euiBorderWidthThin dashed $euiBorderColor;
+  position: absolute !important;  // sass-lint:disable-line no-important
+  margin: 0 !important;  // sass-lint:disable-line no-important
+  bottom: 100%;
+  width: 100%;
+  left: 0;
+  opacity: .9;
+  transform: translate(-12px, 8px);
+  z-index: $euiZLevel2;
+  pointer-events: none;
+  @include euiFocusRing;
```

```suggestion
  box-shadow: 0 0 0 $euiFocusRingSize $euiFocusRingColor;
```

Switching to this will prevent the ghost items from having the focus ring reanimate every time its location changes.